src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
```

Comment:
- Kindly, initialize the book state inside global state as you will have category state also 👍  
```
initialState: {
    books: [],
  },
```